package.json

```diff
@@ -103,5 +103,8 @@
     "rimraf": "latest",
     "ts-jest": "^29.0.0",
     "typescript": "^5.0.0"
+  },
+  "resolutions": {
+    "@comunica/bus-http": "^2.10.0"
```

Comment:
Is this one really needed? I suspect resetting the lockfile should be sufficient to bump the comunica stack.

Reply:
You are totally right 🤦